packages/frontend/src/setupProxy.js

```diff
@@ -1,11 +0,0 @@
-const { createProxyMiddleware } = require('http-proxy-middleware');
```

Comment:
do we not still need this to have the react proxy in dev? or is this handled by vite?

https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually

> Note: You do not need to import this file anywhere. It is automatically registered when you start the development server

Reply:
nice!